config/scorecard/patches/olm.config.yaml

```diff
@@ -0,0 +1,51 @@
+- op: add
+  path: /stages/0/tests/-
+  value:
+    entrypoint:
+    - scorecard-test
+    - olm-bundle-validation
+    image: quay.io/operator-framework/scorecard-test:v1.35.0
+    labels:
+      suite: olm
+      test: olm-bundle-validation-test
+- op: add
+  path: /stages/0/tests/-
+  value:
+    entrypoint:
+    - scorecard-test
+    - olm-crds-have-validation
+    image: quay.io/operator-framework/scorecard-test:v1.35.0
+    labels:
+      suite: olm
+      test: olm-crds-have-validation-test
+# - TODO(macao): This fails since VPA checkpoint does not have resources in CSV. Figure out how to fix this.
```

Comment:
No, I don't think it recommended anything. But AFAIK the checkpoint is simply just a store of historical values? https://povilasv.me/vertical-pod-autoscaling-the-definitive-guide/. I don't really know if that corresponds to concrete k8s resources it could own, so that's why I'm not sure if we should just skip this test.

Reply:
Yeah, having run it, that test is way less great than I had hoped. I think those fields are ultimately advisory? It looks like even if you feed it "pods" or something for the checkpionts, it's also grumbly that the `v1beta2` version of the `VerticalPodAutoscaler` doesn't have any resources either. 

I don't think it matters. I think leaving it commented out for now is perfect.